### PR TITLE
add example workload (with syn attack)

### PR DESCRIPTION
**Pull request by eranra**
The workload is based on GoogleCloudPlatform/microservices-demo
Syn attack is based on docker and an example from https://github.com/bilalcaliskan/syn-flood

### CODE REVIEW

**Comment by eranra** [comment]
> is that something where you can leverage netobserv to spot the attack? how would you demo that? (I'm interested if I can reuse that use-case in demo/talk :) )

Maybe this syn attack traffic is not "seen" by OVS/N ... I need to check more

**Comment by eranra** [comment]
@jpinsonneau I see strange amount of packets (on the edges) when I change the view to show the number of packets::

image

I was expecting a very small amount. And BTW; I see the same in the table ... so something is not making sense here: 

image

**Comment by jotak** [comment]
maybe with sampling = 1?

File: .mk/ocp.mk [hunk 1]
```diff
@@ -8,6 +8,9 @@ ocp-expose:
 	oc expose service loki || true
 	@loki_url=$$(oc get route loki -o jsonpath='{.spec.host}'); \
 	echo -e "\nAccess loki on OCP using: http://"$$loki_url"\n"
+	oc expose -n sample-workload service frontend-external || true
```
Comment (jotak) [inline .mk/ocp.mk:11]: I tend to use `make ocp-expose` just to create a grafana route. Can you split in two different targets then? (maybe `ocp-expose` and `ocp-expose-all`, similarly to the `deploy-all` that deploys more than just the infra ?

Comment (eranra) [reply .mk/ocp.mk:14]: Fixed (@jotak  please review again that this fits your needs )

**Review by jotak — APPROVED** [review]
lgtm!

**Pull request merged**